source/core/src/main/com/csse3200/game/components/tasks/ItemProximityTask.java

```diff
@@ -0,0 +1,107 @@
+package com.csse3200.game.components.tasks;
```

Comment:
The tests are better, I would still like to see ItemProximityTask tests if possible, please (I made some ones like this in the team 4 branch if you needed example tests for tasks)